model/src/main/kotlin/Environment.kt

```diff
@@ -68,7 +68,7 @@ data class Environment(
         /**
          * The version of the OSS Review Toolkit as a string.
          */
-        val ORT_VERSION = this::class.java.getResource("/VERSION").readText()
+        val ORT_VERSION by lazy { this::class.java.`package`.implementationVersion ?: "IDE-SNAPSHOT" }
```

Comment:
What's the idea behind `GRADLE-SNAPSHOT` and `IDE-SNAPSHOT`?

Reply:
When building from the IDE (e.g. when debugging), JARs are not really built / packaged for performance reasons, which means we also cannot access a JAR's metdata here. In that case "IDE-SNAPSHOT" instead of `null` is returned here. Similarly, if a Gradle project's `version` property would not be explicitly set (which actually should never be the case here), Gradle would return "unspecified" as the version, but for consistency with the logic described above I preferred to call this "GRADLE-SNAPSHOT" then instead.

Reply:
I've amended the commit message accordingly.

Reply:
Thanks.